keyboards/dztech/mellow/keyboard.json

```diff
@@ -0,0 +1,298 @@
+{
+    "manufacturer": "dztech",
+    "keyboard_name": "mellow",
+    "maintainer": "moyi4681",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+	"indicators": {
+        "caps_lock": "GP25"
+    },
+    "matrix_pins": {
+        "cols": ["GP29", "GP28", "GP27", "GP26", "GP0", "GP13", "GP12", "GP11", "GP10", "GP9", "GP8", "GP7", "GP6", "GP5"],
+        "rows": ["GP1", "GP2", "GP3", "GP4", "GP24"]
+    },
+    "processor": "RP2040",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0007",
+        "vid": "0x445A"
+    },
+	"community_layouts": [
```

Comment:
Could the following compatible Community Layouts be added:
- 60_ansi_split_bs_split
- 60_ansi_tsangan
- 60_tsangan_hhkb
- 60_ansi_wkl
- 60_ansi_wkl_split_bs_shift
- 60_iso_split_bs_shift
- 60_iso_tsangan
- 60_iso_tsangan_split_bs_shift
- 60_iso_wkl
- 60_iso_wkl_split_bs_shift